src/controllers/auth.js

```diff
@@ -0,0 +1,112 @@
+import db from "../db/index";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcryptjs";
+
+class Authentication {
+
+    static async login(req, res, next) {
+        const { username, password } = req.body;
+
+        try {
+            if (!username || !password) {
+                return res.status(400).json({
```

Comment:
use the response format function in everywhere you need to send HTTP responses

Reply:
now I understand, Thank you